.github/workflows/scheduled-idp-web-tests.yml

```diff
@@ -0,0 +1,156 @@
+name: Scheduled UW IdP Test Matrix
+
+on:
+  push:
+    branches:
+      # You can test this workflow by pushing to this branch
+      - run-scheduled-idp-web-tests
+  schedule:
+    # The cron string must be quoted!
+    # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
+    # Do not change this without also changing stop-test-service-providers.yml
+    - cron: '0 10 * * *'  # 3am PDT, 2am PST
+
+jobs:
+  run-web-tests:
+    strategy:
+      matrix:
+        include:
+          - idp_env: eval
+          - idp_env: prod
```

Comment:
This is the "magic" that runs the test for both environments using the same workflow.